migration/migration.go

```diff
@@ -75,15 +75,16 @@ func createCommon(typeName string, ctx context.Context, witr models.WorkItemType
 	case models.NotFoundError:
 		stString := "string"
 		_, err := witr.Create(ctx, nil, typeName, map[string]app.FieldDefinition{
-			"system.title":       app.FieldDefinition{Type: &app.FieldType{Kind: "string"}, Required: true},
-			"system.description": app.FieldDefinition{Type: &app.FieldType{Kind: "string"}, Required: false},
-			"system.creator":     app.FieldDefinition{Type: &app.FieldType{Kind: "user"}, Required: true},
-			"system.assignee":    app.FieldDefinition{Type: &app.FieldType{Kind: "user"}, Required: false},
+			"system.title":          app.FieldDefinition{Type: &app.FieldType{Kind: "string"}, Required: true},
+			"system.description":    app.FieldDefinition{Type: &app.FieldType{Kind: "string"}, Required: false},
+			"system.creator":        app.FieldDefinition{Type: &app.FieldType{Kind: "user"}, Required: true},
+			"system.assignee":       app.FieldDefinition{Type: &app.FieldType{Kind: "user"}, Required: false},
+			"system.remote_item_id": app.FieldDefinition{Type: &app.FieldType{Kind: "string"}, Required: false},
 			"system.state": app.FieldDefinition{
 				Type: &app.FieldType{
 					BaseType: &stString,
 					Kind:     "enum",
-					Values:   []interface{}{"new", "in progress", "resolved", "closed"},
+					Values:   []interface{}{"new", "open", "in progress", "resolved", "closed"},
```

Comment:
Thanks for tracking it @baijum ,
Did you mean : that we shall add the new state named "open" for now and merge it - and later on introduce the mapping?